x-pack/plugins/apm/public/components/app/service_groups/service_groups_list/service_group_card.tsx

```diff
@@ -83,24 +62,49 @@ export function ServiceGroupsCard({
               )}
           </EuiText>
         </EuiFlexItem>
-        {!hideServiceCount && (
-          <EuiFlexItem>
-            <EuiText size="s">
-              {serviceGroupCounts === undefined ? (
-                <>&nbsp;</>
-              ) : (
-                i18n.translate(
-                  'xpack.apm.serviceGroups.cardsList.serviceCount',
+        <EuiFlexGroup>
+          <ServiceStat loading={isLoading}>
+            <EuiFlexItem>
+              <EuiText size="s" textAlign="left">
+                {serviceGroupCounts === undefined ? (
+                  <>&nbsp;</>
+                ) : (
+                  i18n.translate(
+                    'xpack.apm.serviceGroups.cardsList.serviceCount',
+                    {
+                      defaultMessage:
+                        '{servicesCount} {servicesCount, plural, one {service} other {services}}',
+                      values: { servicesCount: serviceGroupCounts.services },
+                    }
+                  )
+                )}
```

Comment:
Do you need to send`&nbsp`? if not, you can simplify it a bit

```suggestion
                {serviceGroupCounts !== undefined && (
                  i18n.translate(
                    'xpack.apm.serviceGroups.cardsList.serviceCount',
                    {
                      defaultMessage:
                        '{servicesCount} {servicesCount, plural, one {service} other {services}}',
                      values: { servicesCount: serviceGroupCounts.services },
                    }
                  )
                )}
```